packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -315,6 +318,30 @@ bool _checkXcodeVersion() {
   return true;
 }
 
+bool _checkCocoaPodsInstalled() {
+  if (!platform.isMacOS)
+    return false;
+  return exitsHappy(<String>['pod', '--version']);
```

Comment:
Yep, and we won't get here at all if (!platform.isMacOS), so we don't really need to check that.